src/device.rs

```diff
@@ -325,6 +325,11 @@ pub extern "C" fn wgpu_device_create_render_pipeline(
     gfx_select!(device_id => GLOBAL.device_create_render_pipeline(device_id, desc, PhantomData))
 }
 
+#[no_mangle]
+pub extern "C" fn wgpu_device_destroy_render_pipeline(render_pipeline_id: id::RenderPipelineId) {
```

Comment:
could we as well have `wgpu_device_destroy_compute_pipeline` since we are at it?